src/Traits/Http.php

```diff
@@ -342,7 +342,7 @@ protected function validateInputFileField(array $params, $inputFileField)
         }
 
         //All file-paths, urls, or file resources should be provided by using the InputFile object
-        if (! $params[$inputFileField] instanceof InputFile) {
+        if ((!$params[$inputFileField] instanceof InputFile) && (!is_json($params[$inputFileField]))) {
```

Comment:
PHP doesn't have `is_json` method.

Reply:
Yes, I checked my code again and found that it is a function from another library with the following code.

```
function is_json($string)
{
	json_decode($string);
	return (json_last_error() == JSON_ERROR_NONE);
}
```

Reply:
Please add it to the [Validator](https://github.com/irazasyed/telegram-bot-sdk/blob/develop/src/Traits/Validator.php) trait.

Reply:
Thanks.
I've added this change.
Please check it again.